### PR TITLE
Embedded views - Basic wiring with simple views

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Add basic initial support for embedded views.

### Why are these changes necessary?
To support embedded views in flutter.

### How did you verify these changes?
Manual testing.

#### Verification Screenshots:
iOS:
https://github.com/user-attachments/assets/eff35227-a7ed-47a1-b270-b33aa5e31a68

Android:
https://github.com/user-attachments/assets/f9611935-bd9d-4306-a436-79b058d7a67f

### CODE REVIEW

File: android/src/main/kotlin/com/airship/flutter/EmbeddedViewFactory.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.airship.flutter
+
+import android.content.Context
+import io.flutter.plugin.common.BinaryMessenger
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.StandardMessageCodec
+import io.flutter.plugin.platform.PlatformView
+import io.flutter.plugin.platform.PlatformViewFactory
+
+class EmbeddedViewFactory(private val binaryMessenger: BinaryMessenger) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
```
Comment (jyaganeh) [inline android/src/main/kotlin/com/airship/flutter/EmbeddedViewFactory.kt:10]: We should be able to include this class in the other file, like we do in Swift. If flutter doesn't like that for some reason, there's probably a compat annotation that will make it work. lmk if that's the case, I'm happy to help find the right one(s)

File: lib/src/airship_flutter.dart [hunk 2]
```diff
@@ -1,38 +1,51 @@
...
   static final analytics = AirshipAnalytics(_module);
+
   /// The [AirshipActions] instance.
   static final actions = AirshipActions(_module);
+
   /// The [AirshipFeatureFlagManager] instance.
   static final featureFlagManager = AirshipFeatureFlagManager(_module);
 
+  /// The [AirshipAutomation] instance.
+  static final automation = AirshipAutomation(_module);
```
Comment (rlepinski) [inline lib/src/airship_flutter.dart:44]: You should just use the AirshipInApp here

File: lib/src/airship_automation.dart [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  final Map<String, StreamController<bool>> _isEmbeddedAvailableControllers =
+      {};
+  List<EmbeddedInfo> _embeddedInfos = [];
+  late StreamSubscription<EmbeddedInfoUpdatedEvent> _subscription;
+
+  AirshipAutomation(this._module) {
+    _subscription = onEmbeddedInfoUpdated.listen(_updateEmbeddedIds);
+  }
+
+  bool isEmbeddedAvailable({required String embeddedId}) =>
```
Comment (rlepinski) [inline lib/src/airship_automation.dart:42]: If this is async does it cause issues?

File: android/src/main/kotlin/com/airship/flutter/InboxMessageViewFactory.kt [hunk 4]
```diff
@@ -1,17 +0,0 @@
-package com.airship.flutter
-
-import android.content.Context
-import io.flutter.plugin.common.BinaryMessenger
-import io.flutter.plugin.common.MethodChannel
-import io.flutter.plugin.common.StandardMessageCodec
-import io.flutter.plugin.platform.PlatformView
-import io.flutter.plugin.platform.PlatformViewFactory
-
-class InboxMessageViewFactory(private val binaryMessenger: BinaryMessenger) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
```
Comment (rlepinski) [inline android/src/main/kotlin/com/airship/flutter/InboxMessageViewFactory.kt:10]: Why was this removed?

**Comment by rlepinski** [comment]
Android needs the same multicasting changes you made on iOS

**Review by jyaganeh — APPROVED** [review]

File: android/src/main/kotlin/com/airship/flutter/InboxMessageViewFactory.kt (hunk 4 above)
Comment (crow) [reply android/src/main/kotlin/com/airship/flutter/InboxMessageViewFactory.kt:10]: I have reformat on save, just spacing

Comment (crow) [reply android/src/main/kotlin/com/airship/flutter/InboxMessageViewFactory.kt:10]: I just consolidated these the same way the embedded view was consolidated

**Review by rlepinski — APPROVED** [review]
